srcalc/src/main/webapp/WEB-INF/views/displayResults.jsp

```diff
@@ -22,7 +23,20 @@
     </c:forEach>
     </ol>
     <ol>
-    <li><button class="button-em" type="submit" disabled>Sign Calculation</button></li>
+    <li><button id="signCalculationButton" class="button-em" type="submit">Sign Calculation</button>
+        <div class="eSigDialog dialog" title="Enter Electronic Signature Code">
```

Comment:
Putting the dialog inside the button <ol> seems strange. Maybe put it right after the <ol>?